index.html

```diff
@@ -54,7 +54,194 @@ <h1 class="title">
+                <ul class="position">
+                    <li class="name">CANOPY</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="status">Back End Dev</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="date">2015</li>
+                </ul>
+                <p class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
+                    <li class="language"><a
+                            href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javascript</a></li>
+                </ul>
+                <button class="card-button">See Project</button>
+            </div>
+        </div>
+        <div class="card">
+            <img src="images/work4.svg" alt="Project 1" class="card-image">
+            <div class="card-body">
+                <h1 class="card-title">Multi-Post <br> Stories</h1>
+                <ul class="position">
+                    <li class="name">CANOPY</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="status">Back End Dev</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="date">2015</li>
+                </ul>
+                <p class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
+                    <li class="language"><a
+                            href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javascript</a></li>
+                </ul>
+                <button class="card-button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section id="aboutme">
+        <div class="content">
+            <h1 class="title">
+                About
+                <br>
+                Myself
+            </h1>
+            <p class="description">
+                Hello I'm a software developer! I can help you build a product , feature or website Look through some of
+                my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p class="connect-link">
+                LET'S CONNECT
+            </p>
```

Comment:
[ Optional ] Kindly I recommend you to use a header tag hx(h3) If there are other titles (e.g., "Let's connect") instead of `p` tag to make your HTML code semantic correct.

Reply:
Fixed.